src/libs/ajax/data-table-providers/WdsDataTableProvider.test.ts

```diff
@@ -672,7 +672,7 @@ describe('WdsDataTableProvider', () => {
       // ====== Arrange
       const provider = new TestableWdsProvider(uuid, testProxyUrl);
       // ====== Act
-      return provider.importTdr(uuid, uuid).then(() => {
+      return provider.importTdr(uuid, 'manifest.url&snapshotId=anyUuid&other=parameters').then(() => {
         // ====== Assert
         expect(importTdr.mock.calls.length).toBe(1);
```

Comment:
Can anything more specific be asserted here, such as about arguments passed to `importTdr`?